packages/wdio-globals/README.md

```diff
@@ -0,0 +1,22 @@
+WDIO Globals
+============
+
+> A helper utility for importing global variables directly
+
+This package can be used to import global variables explicitly, e.g.
+
+```ts
+import { browser, describe, it } from '@wdio/globals'
```

Comment:
If we can import the `describe` and `it`, can we also import `expect`, `beforeEach` etc?

Reply:
Good catch, `describe` and `it` don't work. It was initially an idea to include them but found out later that it is very difficult to implement this.